build.gradle.kts

```diff
@@ -7,6 +7,7 @@ buildscript {
         mavenCentral()
         google()
         gradlePluginPortal()
+        maven("https://maven.pkg.jetbrains.space/public/p/compose/dev")
```

Comment:
this is not required for compose multiplatform at now as i see in https://kmp.jetbrains.com/

Reply:
without this maven we get error

> Could not resolve all files for configuration ':permissions-compose:kotlinCompilerPluginClasspathIosArm64Main'.
   > Could not find org.jetbrains.compose.compiler:compiler:1.5.15.
     Searched in the following locations:
       - https://repo.maven.apache.org/maven2/org/jetbrains/compose/compiler/compiler/1.5.15/compiler-1.5.15.pom
       - https://dl.google.com/dl/android/maven2/org/jetbrains/compose/compiler/compiler/1.5.15/compiler-1.5.15.pom
     Required by:
         project :permissions-compose